client/verta/verta/client.py

```diff
@@ -732,12 +743,91 @@ def set_repository(self, *args, **kwargs):
 
         """
         return self.get_or_create_repository(*args, **kwargs)
+    # set aliases for get-or-create functions for API compatibility
 
-    def get_or_create_registered_model(self, name=None, desc=None, tags=None, attrs=None, workspace=None, public_within_org=None, id=None):
-        raise NotImplementedError
+    def get_or_create_registered_model(self, *args, **kwargs):
+        """
+        Alias for :meth:`Client.set_project()`.
+
+        """
+        return self.set_registered_model(*args, **kwargs)
 
     def get_registered_model(self, name=None, workspace=None, id=None):
-        raise NotImplementedError
+        name = self._set_from_config_if_none(name, "registered_model")
```

Comment:
This is from a different PR I'm working on. Let's add it here, too, if you don't mind.
```suggestion
        if name is not None and id is not None:
            raise ValueError("cannot specify both `name` and `id`")

        name = self._set_from_config_if_none(name, "registered_model")
        if name is None and id is None:
            raise ValueError("must specify either `name` or `id`")
```